source/Missions/Services/Arena/CoopArenaController.cs

```diff
@@ -92,12 +136,11 @@ private void Handle_JoinInfo(MessagePayload<NetworkMissionJoinInfo> payload)
             Agent newAgent = SpawnAgent(startingPos, joinInfo.CharacterObject, true);
             _agentRegistry.RegisterNetworkControlledAgent(netPeer, joinInfo.PlayerId, newAgent);
 
-            //Mission currentMission = Mission.Current;
 
-            for (int i = 0; i < joinInfo.UnitIdString.Length; i++)
+            for (int i = 0; i < joinInfo?.UnitIdString.Length; i++)

```

Comment:
I don't think this does anything as is
```suggestion
            for (int i = 0; i < joinInfo.UnitIdString?.Length; i++)

```